liferay-lima/liferay-portal
# LPS-182227 Create Tags Dropdown item on D&M

https://issues.liferay.com/browse/LPS-182227


It's under FF  LPS-84424


To test : 
- Go to Documents & Media
- Create files with some tags
- You can see the tags from the group



Problem:
on the figma it's asked this width:

<img width="833" alt="image" src="https://user-images.githubusercontent.com/47524751/233421830-971b9e0e-8e96-4974-a522-dcda3a4f38ee.png">


but if I use the md size :

<img width="862" alt="image" src="https://user-images.githubusercontent.com/47524751/233417495-17188b00-851d-451e-b711-ecd619a07800.png">

I think is something about the 
```
<clay:container-fluid>
	<liferay-ui:search-container
```
From the asset-tag-selector view
And I do not know if is ok as it is now or can be done something ( @jaimelr10 // @ambrinchaudhary )


- LPS-182227 add tags dropdown element
- LPS-182227 create modal with the tags contained
